vispy/app/backends/_tk.py

```diff
@@ -0,0 +1,643 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+vispy backend for Tkinter.
+"""
+
+from __future__ import division
+
+from time import sleep
+
+from ..base import (BaseApplicationBackend, BaseCanvasBackend,
+                    BaseTimerBackend)
+from ...util import keys
+from ...util.ptime import time
+from ... import config
+
+
+# -------------------------------------------------------------------- init ---
+
+# Import
+try:
+    try:
+        import tkinter as tk  # Python >= 3
+    except:
+        import Tkinter as tk  # Python < 3
+    from OpenGL import GL
+    from pyopengltk import OpenGLFrame
+except:
+    available, testable, why_not, which = False, False, "Import error", None
+    
+    class OpenGLFrame(object):
+        pass
+else:
+    available, testable, why_not = True, True, None
+    which = "Tkinter " + str(tk.TkVersion)
```

Comment:
What do you think about providing the tk version *and* the pyopengltk version?

Reply:
I could add `pkg_resources.require("pyopengltk")[0].version`, or make a pull request to [pyopengltk](https://github.com/jonwright/pyopengltk) to add a `__version__` string in their `__init__.py` file.

I's assume the latter is preferred.

Reply:
Yeah, pyopengltk having a version string would be for the best. `pkg_resources` is known to have performance issues when used in large python environments (the more packages installed, the longer it takes). This was a major issue when vispy switched to a new way of describing its version number that included dynamically checking using pkg_resources. For an optional backend this may not be as big of a deal but it would add time to every startup of a Tk based vispy application.

Reply:
I've added a new [PR to pyopengltk](https://github.com/jonwright/pyopengltk/pull/13) to include the version string.
In the meantime, I'll add this to the Tkinter backend:

```python
which_pyopengltk = pyopengltk.__version__ if hasattr(pyopengltk, "__verion__") else "???"
```

Reply:
Looks good to me.